x-pack/plugins/infra/public/pages/logs/log_entry_rate/sections/anomalies/table.tsx

```diff
@@ -240,18 +202,114 @@ export const AnomaliesTable: React.FunctionComponent<{
     ],
     [collapseId, expandId, expandedIds, dateFormat]
   );
+  return (
+    <>
+      <LoadingOverlayWrapper isLoading={isLoading}>
+        <EuiBasicTable
+          items={tableItems}
+          itemId="id"
+          itemIdToExpandedRowMap={expandedDatasetRowContents}
+          isExpandable={true}
+          hasActions={true}
+          columns={columns}
+          sorting={tableSortOptions}
+          onChange={handleTableChange}
+        />
+        <EuiSpacer size="l" />
+        <PaginationControls
+          fetchNextPage={fetchNextPage}
+          fetchPreviousPage={fetchPreviousPage}
+          page={page}
+          isLoading={isLoading}
+        />
+      </LoadingOverlayWrapper>
+    </>
+  );
+};
+
+const AnomalyMessage = ({
+  actual,
+  typical,
+  type,
+}: {
+  actual: number;
+  typical: number;
+  type: string;
+}) => {
+  const messageType = type === 'logRate' ? 'dataset' : 'category';
+  const moreThanExpectedAnomalyMessage = i18n.translate(
+    'xpack.infra.logs.analysis.anomaliesTableMoreThanExpectedAnomalyMessage',
+    {
+      defaultMessage: 'higher log messages in this {messageType} than expected',
+      values: { messageType },
+    }
+  );
```

Comment:
Should the translation include the translations for the types as in

```suggestion
  type: AnomalyType;
}) => {
  const moreThanExpectedAnomalyMessage = i18n.translate(
    'xpack.infra.logs.analysis.anomaliesTableMoreThanExpectedAnomalyMessage',
    {
      defaultMessage: 'higher log messages in this {type, select, logRate {dataset}, logCategory {category}} than expected',
      values: { type },
    }
  );
```

where `AnomalyType` is derived from `anomalyTypeRT`?

Reply:
Are "higher" :left_right_arrow: "fewer" valid opposites?

Reply:
No. To be strict it would be "more" or "greater", but as it's countable and therefore "fewer" and not "less" it should be "more", at least that's what the thesauraus says on the opposite to fewer. (English 😬😬😬).